packages/extras/src/lib/interactivity/setupInteractivity.ts

```diff
@@ -217,12 +237,14 @@ export const setupInteractivity = (context: InteractivityContext) => {
     DOM_EVENTS.forEach(([eventName]) => {
       target.removeEventListener(eventName, getEventHandler(eventName))
     })
+    target.removeEventListener('touchmove', transformTouchMove)
   }
 
   const connect = (target: HTMLElement) => {
     DOM_EVENTS.forEach(([eventName, passive]) => {
       target.addEventListener(eventName, getEventHandler(eventName), { passive })
     })
+    target.addEventListener('touchmove', transformTouchMove, { passive: true })
```

Comment:
Why don't I just add touchmove to DOM_EVENTS? TouchEvent has a very different interface from PointerEvent, so adding it to the `type DomEvent = PointerEvent | MouseEvent | WheelEvent` union would require a lot of downstream changes including a breaking change to users since the native event is passed along